package.json

```diff
@@ -44,7 +44,7 @@
   "dependencies": {
     "@jakechampion/wizer": "^1.6.0",
     "esbuild": "^0.15.16",
-    "js-component-tools": "0.2.2",
+    "js-component-tools": "0.3.0-beta.2",
```

Comment:
question: do we know when this is coming out of beta?

Reply:
It's purely a js-component-tools thing for now, and there's no reason not to already, probably going to give things another couple of weeks to settle.